### DISCUSSION TITLE
Snippets

### DISCUSSION THREAD

**Discussion by yadoga** [zed-industries/zed#7832] [Feature Requests]
Unsure if snippets are available yet. I've seen some people showing custom solutions like
 ```
  "completion": {
      "snippets": {
          "custom": {
```
which I had no success with. 

Looking for tips on how to incorporate snippets in Svelte LSP (and globally). Could somebody share their settings.json here? 🙌

**Comment by Moshyfawn** [comment]
There's #4611 for global snippets and #5743, which was the original snippets issue, which seems to have some snippet examples in the thread.
